lab_1/main.py

```diff
@@ -138,26 +144,38 @@ def get_adjacent_words(tokens: list, word: str, left_n: int, right_n: int) -> li
     right_n = 3
     --> [['man', 'is'], ['dog, 'cat']]
     """
-    pass
-
-
-
+    concordance = get_concordance(tokens, word, left_n, right_n)
+    adj_words = []
+    for el in concordance:
+        if left_n > 0 and right_n > 0:
+            adj_words.append([el[0], el[-1]])
+        elif left_n > 0:
+            adj_words.append([el[0]])
+        elif right_n > 0:
+            adj_words.append([el[-1]])
+    return adj_words
 def read_from_file(path_to_file: str) -> str:
     """
     Opens the file and reads its content
     :return: the initial text in string format
     """
     with open(path_to_file, 'r', encoding='utf-8') as fs:
         data = fs.read()
-
+    with open(path_to_file, 'r', encoding='utf-8') as file:
+        data = file.read()
     return data
 
 
 def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    if isinstance(path_to_file, str) and isinstance(content, list):
+        with open(path_to_file, 'w', encoding='utf-8') as file:
+            for i in content:
+                file.write(" ".join(i) + "\n")
```

Comment:
put all the lines in one variable
then write this variable to file

it's not good to write to file at each iteration